README.md

```diff
@@ -60,13 +60,23 @@ $ vim templates/logsearch-for-cf.example-with-uaa-auth.yml
 $ scripts/generate_deployment_manifest ~/workspace/logsearch.yml templates/logsearch-for-cf.example-with-uaa-auth.yml > ~/workspace/logsearch-with-logsearch-for-cf.yml
 ```
 
-### 3. Update the logsearch deployment with the new manifest
+### 4. Update the logsearch deployment with the new manifest
 
 ```sh
 $ bosh deployment ~/workspace/logsearch-with-logsearch-for-cf.yml
 $ bosh deploy
 ```
 
+### 5. Update Cloud Foundry deployment to forward component logs to ingestor
+
+```yaml
+properties:
+  syslog_daemon_config:
+    address: <ingestor-static-ip>
```

Comment:
Now that I think of it, it probably makes more sense to reference the haproxy job here so that we load balance across ingestors if we have more than one: `0.ls-router.default.logsearch.bosh`.